public/js/index.js

```diff
@@ -128,6 +128,17 @@ $(function () {
         console.error("Error saving config:", strError);
       }
     });
+
+  })
+
+  $('#updateSoftware').on('click', function () {
+    if (objSocket.connected) {
+      objSocket.emit('control_hat', '{“set_charger”: “ON”}')
+      objSocket.emit('update', `{"timestamp":"${Date.now()}", "version":"${RQ_PARAMS.UPDATE_VERSION}", "action":"UPDATE", "args":"UI"}`)
```

Comment:
The updater.py script has an internal constant named UPDATE_VERSION. 
The NodeJS code has RQ_PARAMS.CONFIG_FORMAT_VERSION.

To reduce confusion, I believe a better name here is UPDATE_FORMAT_VERSION.

Reply:
done